server/apiRoutes/post.js

```diff
@@ -58,5 +72,32 @@ router.get('/:postId', async (req, res, next) => {
     }
 });
 
+router.delete('/deletePost/:postId', isLoggedIn, async (req, res, next) => {
+    if (req.user.isSiteAdmin) {
+        const deleted = await Post.destroy({
+            where: {
+                id: req.params.id,
+            }
+        });
+        if (deleted) {
+            res.status(201);
+        } else {
+            res.status(401);
+        }
+    } else {
```

Comment:
This seems weird. I think you should query for the instance. Then check the userId on the instance, if they are equal, delete the instance.

Reply:
The reason behind why I have both is for a user trying to delete their own post, or an admin forcefully deleting a post.